### PR TITLE
Closes BG-1107: Splits standalone step

### PR DESCRIPTION
**Pull request by ap-justin**
## Explanation of the solution
* create standalone splits step
image

*  remove `<AdvanceOptions/>` from donate forms 
sample: Crypto form
image

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
   "devDependencies": {
     "@biomejs/biome": "1.5.3",
     "@keplr-wallet/types": "0.12.23",
+    "@radix-ui/react-slider": "1.1.2",
```
Comment (ap-justin) [inline package.json:73]: [radix-ui slider](https://www.radix-ui.com/primitives/docs/components/slider) replaces native `<input type="range"/>`  with lots of css overrides to achieve style requirement

File: src/components/TokenField/TokenField.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ export default function TokenField<T extends FieldValues, K extends Path<T>>({
           id="amount"
           type="text"
           placeholder="0.0000"
-          className="text-sm py-3 dark:text-gray"
+          className="text-sm py-3.5 dark:text-gray"
```
Comment (ap-justin) [inline src/components/TokenField/TokenField.tsx:67]: adjust to match `<DonationAmount/>` spacing of striep and paypal

File: src/components/donation/Steps/DonateMethods/DonateMethods.tsx [hunk 3]
```diff
@@ -85,7 +85,7 @@ export default function DonateMethods({ donaterConfig, state }: Props) {
           </button>
         )}
       </Tab.List>
-      <Tab.Panels as="div" className="p-4 @md:p-8 pt-0 @md:pt-4 ">
+      <Tab.Panels as="div" className="grid p-4 @md:p-8 pt-0 @md:pt-4 ">
```
Comment (ap-justin) [inline src/components/donation/Steps/DonateMethods/DonateMethods.tsx:88]: so that child donate forms would consume full tab panel height

**Review by misicnenad — COMMENTED** [review]
Current code looks good, although it's still needs to address how the widget UI
https://linear.app/better-giving/issue/BG-1107/move-split-to-standalone-step#comment-ec4d160a

**Review by SovereignAndrey — APPROVED** [review]
Looks great! :100:

**Pull request merged**